### PR TITLE
MN-532: optimization topsyncpulse

### PR DESCRIPTION
**Pull request by sansaian**
https://insolar.atlassian.net/browse/MN-532

### CODE REVIEW

File: scripts/monitor/grafana/dashboards/observer.json [hunk 1]
```diff
@@ -719,8 +753,8 @@
     ]
   },
   "time": {
-    "from": "now-30m",
-    "to": "now"
+    "from": "2020-05-29T16:00:45.933Z",
```
Comment (pavlushev) [inline scripts/monitor/grafana/dashboards/observer.json:756]: It is correct?

File: component/fetcher.go [hunk 2]
```diff
@@ -51,12 +51,15 @@ func makeFetcher(
 		recordCounterMetric.Add(float64(len(batch)))
 		log.WithField("batch_size", len(batch)).
 			Infof("fetched records")
-
+		if pulse.Number < s.currentHeavyPN {
+			return &raw{pulse: pulse, batch: batch, shouldIterateFrom: shouldIterateFrom}
```
Comment (megge-dream) [inline component/fetcher.go:55]: Why we don't set `currentHeavyPN` here?

Comment (SergeyRadist) [reply component/fetcher.go:55]: yep, you need to return raw.currentHeavyPN to fastForwardInterwal works properly

Comment (sansaian) [reply component/fetcher.go]: Done

File: scripts/monitor/grafana/dashboards/observer.json (hunk 1 above)
Comment (sansaian) [reply scripts/monitor/grafana/dashboards/observer.json]: return old value

**Pull request merged**